pkg/artifactory/resource_artifactory_remote_repository.go

```diff
@@ -290,6 +314,14 @@ func unpackRemoteRepo(s *schema.ResourceData) *v1.RemoteRepository {
 	repo.VcsGitProvider = d.getStringRef("vcs_git_provider")
 	repo.VcsType = d.getStringRef("vcs_type")
 	repo.XrayIndex = d.getBoolRef("xray_index")
+	if v, ok := d.GetOk("nuget"); ok {
+		nugetConfig := v.(*schema.Set).List()[0].(map[string]interface{})
+		repo.Nuget = &v1.Nuget{
+			FeedContextPath:     &nugetConfig["feed_context_path"],
```

Comment:
What is the behaviour here if the map does not contain the key? Could this be taking the address of nil?

Reply:
My understanding (which is quite probably wrong), is that it'd be set to the default value. I'll do some testing tomorrow to find out